docs/reference_guide.md

```diff
@@ -99,7 +99,7 @@ This instruments the tcp_v4_connect() kernel function using a kprobe, with the f
 - ```struct pt_regs *ctx```: Registers and BPF context.
 - ```struct sock *sk```: First argument to tcp_v4_connect().
 
-The first argument is always ```struct pt_regs *```, the remainder are the arguments to the function (they don't need to be specified, if you don't intend to use them).
+The first argument is always ```struct pt_regs *```, the remainder are the arguments to the function (they don't need to be specified, if you don't intend to use them). Limitations: maximum of 4 args including ctx.
```

Comment:
It should be higher, 6 + ctx is what the rewriter supports. The test for overflow is in https://github.com/iovisor/bcc/blob/master/tests/python/test_clang.py#L345